### PR TITLE
Stop people in refugee center from bashing and tell them to return to places.

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
Bugfixes "Adds NO_BASH and RETURN_TO_START_POS to most NPCs in refugee center."

#### Purpose of change

Partially addresses  #62470

#### Describe the solution

Gives all the surface refugees and guards the listed traits, but allows them to still investigate sounds. I may give some of the quest giving NPCs the no investigate flag as well.

Created a novel guard class for the refugee center in the process rather than overriding traits. This gives them all an evac jacket which should be nice and uniform. These are the interior guards at the center, and so I have assumed these folks have minimal training and are using melee weapons, not guns. Later, in Experimental, I'd like to propose a quest line to help them improve their training and equipment.

#### Describe alternatives you've considered

As above - some of them could probably get no investigate as well.

#### Testing
TBA - at work.

### CODE REVIEW

File: data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard1.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
     "id": "evac_guard1",
     "//": "Appears in the refugee center as a guard with custom dialogue.",
     "name_suffix": "Guard",
-    "class": "NC_BOUNTY_HUNTER",
+    "class": "NC_FREE_MERCH_GUARD",
+    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
```
Comment (I-am-Erk) [inline data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard1.json:8]:
```suggestion
    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
    "worn_override": "FREE_MERCH_GUARD_worn_phase1",
```

File: data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard2.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
     "id": "evac_guard2",
     "//": "Appears in the refugee center as a guard with custom dialogue.",
     "name_suffix": "Guard",
-    "class": "NC_BOUNTY_HUNTER",
+    "class": "NC_FREE_MERCH_GUARD",
+    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
```
Comment (I-am-Erk) [inline data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard2.json:8]:
```suggestion
    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
    "worn_override": "FREE_MERCH_GUARD_worn_phase1",
```

File: data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard3.json [hunk 3]
```diff
@@ -4,7 +4,8 @@
     "id": "evac_guard3",
     "//": "Appears in the refugee center as a guard with custom dialogue.",
     "name_suffix": "Guard",
-    "class": "NC_BOUNTY_HUNTER",
+    "class": "NC_FREE_MERCH_GUARD",
+    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
```
Comment (I-am-Erk) [inline data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard3.json:8]:
```suggestion
    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
    "worn_override": "FREE_MERCH_GUARD_worn_phase1",
```

File: data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard_generic.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
     "id": "guard",
     "//": "A generic guard for the Free Merchants faction.",
     "name_suffix": "Guard",
-    "class": "NC_BOUNTY_HUNTER",
+    "class": "NC_FREE_MERCH_GUARD",
+    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
```
Comment (I-am-Erk) [inline data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_guard_generic.json:8]:
```suggestion
    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
    "worn_override": "FREE_MERCH_GUARD_worn_phase1",
```

File: data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_hostile_guard.json [hunk 5]
```diff
@@ -4,7 +4,8 @@
     "id": "hostile_guard",
     "//": "A generic hostile guard for the Free Merchants faction.  For where the player shouldn't venture.'",
     "name_suffix": "Guard",
-    "class": "NC_BOUNTY_HUNTER",
+    "class": "NC_FREE_MERCH_GUARD",
+    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
```
Comment (I-am-Erk) [inline data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_hostile_guard.json:8]:
```suggestion
    "weapon_override": "FREE_MERCH_GUARD_wield_phase1",
    "worn_override": "FREE_MERCH_GUARD_worn_phase1",
```

**Comment by MNG-cataclysm** [comment]
Apparently, the new guard class you created is considered invalid by the tests.

File: data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_teamster.json [hunk 6]
```diff
@@ -8,7 +8,8 @@
     "attitude": 0,
     "mission": 3,
     "chat": "TALK_FREE_MERCHANT_TEAMSTER",
-    "faction": "free_merchants"
+    "faction": "free_merchants",
+    "traits": [ { "trait": "IGNORE_SOUND" }, { "trait": "NO_BASH" }, { "trait": "RETURN_TO_START_POS" } ]
```
Comment (MNG-cataclysm) [inline data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_teamster.json:12]: You need to move the traits to the NPC Class definition. They aren’t registered here.

**Review by MNG-cataclysm — CHANGES_REQUESTED** [review]

File: data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_teamster.json (hunk 6 above)
Comment (I-am-Erk) [reply data/json/npcs/refugee_center/surface_staff/NPC_free_merchant_teamster.json:12]: I'm aware. Just been too burnt out from work to fix it for a few days.

**Comment by MNG-cataclysm** [comment]
As #63289 merged the content from this PR, it should probably be closed.

**Comment by Maleclypse** [comment]
Completed by #63289

**Pull request closed without merging**